Json.More/JsonElementProxy.cs

```diff
@@ -49,6 +50,8 @@ private JsonElementProxy(JsonElement value)
 	/// Converts an `string` to a <see cref="JsonElementProxy"/>.
 	/// </summary>
 	/// <param name="value">The value.</param>
+	[RequiresDynamicCode("Calls Json.More.JsonElementExtensions.AsJsonElement(string)")]
```

Comment:
These attributes can be removed (they no longer call something that's annotated RDC/RUC).